vocabularies/survey-type.ttl

```diff
@@ -165,12 +174,14 @@ svytp:process a skos:Concept ;
         "resampling"@en ;
     skos:definition "A sampling procedure performed on an existing sample to alter the sample or generate a new sub-sample or set of sub-samples. This may include seismic processing and reprocessing."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "ps" ;
     skos:prefLabel "Process"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 
 svytp:atmospheric a skos:Concept ;
     skos:definition "A survey measuring chemical and physical properties of the atmosphere."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "at" ;
     skos:prefLabel "Atmospheric"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 

```

Comment:
Are these electrical altLabels appropriate, or should they be methods instead?
@KellyVance @dxwell @DavidCrosswellGSQ

Reply:
yeah. I think they are methods. should migrate them soon